src/components/Postblock.vue

```diff
@@ -0,0 +1,126 @@
+<script setup>
+</script>
+
+<template>
+    <div class="top_image_post_block">
+      <img class="main_img_post_block" src="../assets/postphoto.jpg" alt="post image">
+    <div class="info_post_block">
+      <div class="header_info_post_block"><a src="#">PHOTOGRAPHY</a></div>
+      <div class="name_info_post_block"><a src="#">5 REASONS YOU SHOULD FALL IN LOVE WITH PHOTOGRAPHY</a></div>
+      <div class="date_info_post_block"><a src="#">May 26, 2016 - by Martin Vegas</a></div>
+      <div class="main_text_info_post_block">Suspendisse imperdiet ante at tortor consequat consectetur. Quisque rhoncus blandit justo. Praesent congue convallis erat, utem dignissim...</div>
+      <div class="dots_info_post_block"><a src="#">...</a></div>
+    </div>
+  </div>
+</template>
+
+<style scoped>
+template{
```

Comment:
do not stale on template bur rather on div component which is wrapping your whole content in component